src/test/java/codeu/controller/LoginServletTest.java

```diff
@@ -60,53 +63,22 @@ public void testDoGet() throws IOException, ServletException {
 
     Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
   }
-
```

Comment:
Guessing this was an accidental deletion? Must be that your repo wasn't synced to the the head of master when you pushed :(